package.json

```diff
@@ -185,7 +185,7 @@
     "react-native-web-linear-gradient": "^1.1.2",
     "react-native-web-sound": "^0.1.3",
     "react-native-webview": "13.8.6",
-    "react-pdf": "^7.7.3",
+    "react-pdf": "9.1.0",
```

Comment:
We can keep or discard the changes related to `pdf`. The reason is that [here](https://github.com/Expensify/App/issues/48327#issuecomment-2323039198) we already have a tracker to update `react-pdf`. If we decide to keep the changes, the linked issue probably won't have any affect because it will be deployed in the newer versions, so they can do a bump whenever that issue is fixed.

Once that happens we will have to update `react-fast-pdf` as well because it depends on `react-pdf` and we no longer have `pdfjs-dist/legacy/build` instead we have `pdfjs-dist/build`. This is also explained in the issue [here](https://github.com/Expensify/App/issues/48327). I have added a patch for `react-fast-pdf` in this PR thinking that we can first get this PR merged with the patch and then follow up on `react-fast-pdf` by fixing the issue there and bumping and removing the patch in `Expensify App`.